[2024-03-27T15:50:06.700+0200] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_id_3.show_databases manual__2024-03-26T12:56:14.482903+00:00 [queued]>
[2024-03-27T15:50:06.733+0200] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_id_3.show_databases manual__2024-03-26T12:56:14.482903+00:00 [queued]>
[2024-03-27T15:50:06.734+0200] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-03-27T15:50:06.757+0200] {taskinstance.py:2214} INFO - Executing <Task(PostgresOperator): show_databases> on 2024-03-26 12:56:14.482903+00:00
[2024-03-27T15:50:06.764+0200] {standard_task_runner.py:60} INFO - Started process 57540 to run task
[2024-03-27T15:50:06.774+0200] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'dag_id_3', 'show_databases', 'manual__2024-03-26T12:56:14.482903+00:00', '--job-id', '713', '--raw', '--subdir', 'DAGS_FOLDER/jobs_dag.py', '--cfg-path', '/var/folders/t6/vwjsg0jx26jcjdwn_gxhbjzc0000gp/T/tmpf5qk6iak']
[2024-03-27T15:50:06.778+0200] {standard_task_runner.py:88} INFO - Job 713: Subtask show_databases
[2024-03-27T15:50:06.996+0200] {task_command.py:423} INFO - Running <TaskInstance: dag_id_3.show_databases manual__2024-03-26T12:56:14.482903+00:00 [running]> on host C9843
[2024-03-27T15:50:07.282+0200] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='dag_id_3' AIRFLOW_CTX_TASK_ID='show_databases' AIRFLOW_CTX_EXECUTION_DATE='2024-03-26T12:56:14.482903+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-26T12:56:14.482903+00:00'
[2024-03-27T15:50:07.285+0200] {sql.py:276} INFO - Executing: 
                SELECT datname
                FROM pg_database
                WHERE datistemplate = false
                ORDER BY datname;
            
[2024-03-27T15:50:07.303+0200] {base.py:83} INFO - Using connection ID 'Postgres_db' for task execution.
[2024-03-27T15:50:07.400+0200] {base.py:83} INFO - Using connection ID 'Postgres_db' for task execution.
[2024-03-27T15:50:07.437+0200] {sql.py:457} INFO - Running statement: 
                SELECT datname
                FROM pg_database
                WHERE datistemplate = false
                ORDER BY datname;
            , parameters: [51000271, 'abatcoveanu', datetime.datetime(2024, 3, 27, 15, 50, 5, 93751)]
[2024-03-27T15:50:07.449+0200] {sql.py:466} INFO - Rows affected: 3
[2024-03-27T15:50:07.464+0200] {logging_mixin.py:188} WARNING - /usr/local/lib/python3.8/site-packages/airflow/providers/common/sql/hooks/sql.py:407 AirflowProviderDeprecationWarning: Call to deprecated method _make_common_data_structure. (The `_make_serializable` method is deprecated and support will be removed in a future version of the common.sql provider. Please update the DbApiHook's provider to a version based on common.sql >= 1.9.1.)
[2024-03-27T15:50:07.603+0200] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=dag_id_3, task_id=show_databases, execution_date=20240326T125614, start_date=20240327T135006, end_date=20240327T135007
[2024-03-27T15:50:07.680+0200] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-27T15:50:07.729+0200] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
